.pipelines/dev-tunnels-ssh-ci.yaml

```diff
@@ -168,7 +185,7 @@ jobs:
         continueOnError: True
         inputs:
           command: push
-          searchPatternPush: out/pkg/*.nupkg
+          searchPatternPush: out/pkg/*.nupkg;out/pkg/*.snupkg
```

Comment:
Although I'm not sure how to test this without actually publishing the package to nuget, I am reasonably certain this is the correct command based on these docs:

- [publishing a symbol package](https://learn.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg#publishing-a-symbol-package)
- [NuGetCommand@2](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/nuget-command-v2?view=azure-pipelines)